sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/ErrorCodeValue.java

```diff
@@ -8,7 +8,7 @@
 import com.fasterxml.jackson.annotation.JsonValue;
 
 /**
- * Defines values for ErrorCodeValue.
+ * Defines values for TextAnalyticsErrorCode.
  */
 public enum ErrorCodeValue {
```

Comment:
We have error code as a string in .NET rather than having an enum for it.  Not sure what the philosophy is on this on the Java side, but in .NET if we did expose this as an enum, we would want a service-specific name for it to avoid type name clashes down the line.  This is a candidate for a common library, but we're still operating under the assumption that we won't get there for TextAnalytics (this could change).

Reply:
Correct, this was reverted as it was a code generated file and should not have been changed. 
The enum exposed to the users is named `TextAnalyticsErrorCode`.

Reply:
Ah, ok.  I must have read the code incorrectly.

Reply:
Do you have the same issue that we have in .NET where it's a breaking change to add values to an enum?

Reply:
No, it wouldn't since in Java we use [ExpandableStringEnum](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/src/main/java/com/azure/core/util/ExpandableStringEnum.java) that allows this enum to accept any future values that service team could add.

Reply:
>ExpandableStringEnum that allows this enum to accept any future values that service team could add.

Cool :)